packages/story-editor/src/components/richText/karma/singleSelection.karma.js

```diff
@@ -163,6 +161,7 @@ describe('Styling single text field', () => {
       await data.fixture.events.sleep(300);
       await data.fixture.events.click(fontColor.button);
       waitFor(() => fontColor.picker);
+      await data.fixture.events.click(fontColor.picker.custom);
```

Comment:
Should we add some Karma tests for the basic color picker as well? I've approved regardless since you've covered that part with unit tests, so feel free to ignore.